### DISCUSSION TITLE
p:panelgrid: facet:header causes colums to be merged

### DISCUSSION THREAD

**Discussion by FranzHagel** [primefaces/community#990] [PrimeFaces] [answered]
Using PF 13.05, jsf-impl 2.3.14.SP07.

The following code renders one DIV for two columns:
```
<p:panelGrid id="BudgetInfo">
	<f:facet name="header">
		<p:row>
			<p:column style="width: 25vw;">Sparte</p:column>
			<p:column style="width: 25vw;">Budget</p:column>
		</p:row>
	</f:facet>
</p:panelGrid>
```
![image](https://github.com/primefaces/community/assets/40021634/fa0e4da0-75d0-4b1c-ae58-4ecca8bc28b7)

Without "f:facet" it is rendered correctly as two columns:
```
<p:panelGrid id="BudgetInfo">
	<p:row>
		<p:column style="width: 25vw;">Sparte</p:column>
		<p:column style="width: 25vw;">Budget</p:column>
	</p:row>
</p:panelGrid>
```

**Comment by melloware** [comment, marked as answer]
Yes the header facet is not designed for columns and rows its just a header.  Only the content accepts columns and rows

**Comment by FranzHagel** [comment]
It works in PF 12. In the showcase of PF 13 the facet is used with rows and columns in the example for the tabular layout:

```
  <p:panelGrid style="margin-top:20px" layout="tabular" columns="0">
        <f:facet name="header">
            <p:row>
                <p:column colspan="7">1995-96 NBA Playoffs</p:column>
            </p:row>
```

**Reply by melloware** [reply]
Ok can you open a github issue then please if you said it used to work. Make sure to include reproducer

**Reply by melloware** [reply]
Actually check the PF 13 migration guide there is a note in there for PanelGrid

**Comment by FranzHagel** [comment]
Thanks!
